packages/excalidraw/data/library.ts

```diff
@@ -97,19 +162,28 @@ class Library {
     if (this.updateQueue.length > 0) {
       jotaiStore.set(libraryItemsAtom, {
         status: "loading",
-        libraryItems: this.lastLibraryItems,
+        libraryItems: this.currLibraryItems,
         isInitialized: this.isInitialized,
       });
     } else {
       this.isInitialized = true;
       jotaiStore.set(libraryItemsAtom, {
         status: "loaded",
-        libraryItems: this.lastLibraryItems,
+        libraryItems: this.currLibraryItems,
         isInitialized: this.isInitialized,
       });
       try {
-        this.app.props.onLibraryChange?.(
-          cloneLibraryItems(this.lastLibraryItems),
+        const prevLibraryItems = this.prevLibraryItems;
+        this.prevLibraryItems = cloneLibraryItems(this.currLibraryItems);
+
+        const nextLibraryItems = cloneLibraryItems(this.currLibraryItems);
+
+        const change = createLibraryChange(prevLibraryItems, nextLibraryItems);
+
+        this.app.props.onLibraryChange?.(nextLibraryItems, change);
```

Comment:
`nextLibraryItems` are now duplicated? (as part of the first param and the change.inserted)

Reply:
correct. The idea was that we'll slowly (not sure how) move to emitting only `change`. Moot now as I've removed the `change` arg from public API as per your suggestion.